docs/userguide/searchspace.md

```diff
@@ -1,46 +1,269 @@
+]
+subspace = SubspaceDiscrete.from_simplex(max_sum=1.0, simplex_parameters=parameters)
+```
+
+~~~
+       p1    p2
+ 0   0.00  0.00
+ 1   0.00  0.25
+ 2   0.00  0.50
+ ..   ...   ...
+ 12  0.75  0.00
+ 13  0.75  0.25
+ 14  1.00  0.00
+ 
+ [15 rows x 2 columns]
+~~~
+
+### Representation of data within discrete subspaces
+
+Internally, discrete subspaces are represented by two dataframes, the *experimental* and the *computational* representation.
+
+The experimental representation (`exp_rep`) contains all parameters as they were provided upon the construction of the search space and viewed by the experimenter. The computational representation (`comp_rep`) contains a representation of parameters that is actually used for the internal calculation.
+
+In particular, the computational representation contains no more labels or constant columns. This happens e.g. for [`SubstanceParameter`](baybe.parameters.substance.SubstanceParameter) or [`CategoricalParameter`](baybe.parameters.categorical.CategoricalParameter). Further note that the shape of the computational representation can also change depending on the chosen encoding.
+
+The following example demonstrates the difference:
+```python
+from baybe.parameters import NumericalDiscreteParameter, CategoricalParameter
+from baybe.searchspace import SearchSpace
+
+speed = CategoricalParameter("Speed", values=["slow", "normal", "fast"], encoding="OHE")
+temperature = NumericalDiscreteParameter(name="Temperature", values=[90, 105])
+
+searchspace = SearchSpace.from_product(parameters=[speed, temperature])
+```
+
+~~~
+  Experimental Representation
+      Speed  Temperature
+  0    slow         90.0
+  1    slow        105.0
+  2  normal         90.0
+  3  normal        105.0
+  4    fast         90.0
+  5    fast        105.0
+
+  Computational Representation
+     Speed_slow  Speed_normal  Speed_fast  Temperature
+  0           1             0           0         90.0
+  1           1             0           0        105.0
+  2           0             1           0         90.0
+  3           0             1           0        105.0
+  4           0             0           1         90.0
+  5           0             0           1        105.0
+~~~
+
+### Metadata
+
+```{warning}
+Although possible, it is not intended to manually change the metadata. The metadata is maintained internally, and there is a risk involved with manipulating it manually.
+Consequently, we advise to only change the metadata manually if you are certain about it.
+```
+
+Discrete subspaces keep track of the recommendations that were made during a campaign.
+The information is stored in a `metadata` field, and it is possible to manually modify `metadata` to influence the behavior of the corresponding `campaign` exploring the space. In particular, by manually changing the values of `metadata["dont_recommend"]`, it is possible to prevent certain points of the subspace from being recommended.
```

Comment:
insteadd of just `metadata` Id would at least once mention the full access `searchspace.discrete.metadata`